test/e2e-test/ci-test.sh

```diff
@@ -0,0 +1,140 @@
+apt-get update
+apt-get install helm
+
+#Create a private repository
+if [ "$(docker inspect -f '{{.State.Running}}' "${reg_name}" 2>/dev/null || true)" != 'true' ]; then
+  docker run \
+    -d --restart=always -p "127.0.0.1:${reg_port}:5000" --name "${reg_name}" \
+    registry:2
+fi
+
+
+#Initialize a kind-K8S cluster and add the private repo to the configuration file
+kind create cluster --name $cluster_name  --config $kind_config
+
+REGISTRY_DIR="/etc/containerd/certs.d/localhost:${reg_port}"
+for node in $(kind get nodes --name $cluster_name); do
+  docker exec "${node}" mkdir -p "${REGISTRY_DIR}"
+  cat <<EOF | docker exec -i "${node}" cp /dev/stdin "${REGISTRY_DIR}/hosts.toml"
+[host."http://${reg_name}:5000"]
+EOF
+done
+if [ "$(docker inspect -f='{{json .NetworkSettings.Networks.kind}}' "${reg_name}")" = 'null' ]; then
+  docker network connect "kind" "${reg_name}"
+fi
+kubectl apply -f $repo_configmap
+
+
+
+#Build and push images
+cd  ../../device-plugin/ccnp-device-plugin/
+add_proxy container/Dockerfile
+docker build -t $device_plugin -f container/Dockerfile .
+cd  ../..
+add_proxy container/quote-server/Dockerfile
+docker build -t $get_quote -f container/quote-server/Dockerfile .
+add_proxy container/measurement-server/Dockerfile
+docker build -t $measurement -f container/measurement-server/Dockerfile .
+add_proxy container/eventlog-server/Dockerfile
+docker build -t $eventlog -f container/eventlog-server/Dockerfile .
+
+docker push ${device_plugin}
+docker push ${get_quote}
+docker push ${measurement}
+docker push ${eventlog}
+
+#Deploy CCNP Dependencies
+helm repo add nfd https://kubernetes-sigs.github.io/node-feature-discovery/charts
+helm repo update
+helm install nfd/node-feature-discovery --namespace $nfd_ns --create-namespace --generate-name
+kubectl apply -f  device-plugin/ccnp-device-plugin/deploy/node-feature-rules.yaml
+change_value $docker_repo  $local_repo  device-plugin/ccnp-device-plugin/deploy/helm/ccnp-device-plugin/values.yaml
+helm install ccnp-device-plugin  device-plugin/ccnp-device-plugin/deploy/helm/ccnp-device-plugin
+
+#Deploy CCNP services
+kubectl create -f deployment/manifests/namespace.yaml
+change_value $docker_repo  $local_repo  deployment/manifests/eventlog-server-deployment.yaml
+kubectl create -f deployment/manifests/eventlog-server-deployment.yaml
+change_value $docker_repo  $local_repo  deployment/manifests/measurement-server-deployment.yaml
+kubectl create -f deployment/manifests/measurement-server-deployment.yaml
+change_value $docker_repo  $local_repo  deployment/manifests/quote-server-deployment.yaml
+kubectl apply -f deployment/manifests/quote-server-deployment.yaml
+
+#Install SDK
+cd sdk/python3/
+pip install -r requirements.txt
+pip install -e .
+pip3 install pytest
+pip install pytdxattest
+cd ../../..
+sleep 2m
+
+#Run test cases
+pytest  test_eventlog.py test_tdquote.py  test_tdreport.py
+#Clean 
+echo y | ./ci-clean.sh > /dev/null
```

Comment:
no need to include the clean script here, this should be done in github action workflow yaml.

Reply:
Fixed.